### ISSUE TITLE
Add useful error pages

### ISSUE THREAD

**Issue by dvidsilva** [TechForPR/chefsForPR#43]
At least for 404, right now it doesn't show any useful information, a simple empty page with the layout, logo and link to homepage would be great.

**Comment by apravink** [comment]
I would like to give this one a shot. Just to make sure I am headed in the right direction, you would like to add more info to the 404 template and possibly the error template?.

Could you specify a little more about the layout?

**Comment by dvidsilva** [comment]
Yes, this is it. You can modifiy it so it looks better, and add better error messages. Is currently kinda working 

The code is around `index.js`, like:

```
app.use(function(err, req, res) {
  // set locals, only providing error in development
  res.locals.message = err.message;
  res.locals.error = req.app.get('env') === 'development' ? err : {};

  // render the error page
  res.status(err.status || 500);
  res.render('error');
});
```

There's a few answers in SO like [this](https://stackoverflow.com/questions/7641622/node-js-express-render-error-when-page-not-found), but I think something like this code from `routes/index.js` might be enough:

```
 return res.status(404).render('404');
```

@apravink
